src/room.cpp

```diff
@@ -0,0 +1,188 @@
+// Copyright 2017 Tunnler Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#include "tunnler/room.h"
+
```

Comment:
I get that - but this is something lioncash (= LC) would say during review (and LC reviews *everything*).
He's very "uptight" (for a lack of a better term) about keeping code short by removing useless empty lines :P